recipes/openssl/1.x.x/test_package/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ set(_custom_vars
     OPENSSL_VERSION
 )
 foreach(_custom_var ${_custom_vars})
-    if(DEFINED _custom_var)
+    if(DEFINED ${_custom_var})
```

Comment:
I think it is correct as it is

https://stackoverflow.com/a/51621325/2207935

Reply:
> (unless it contains the name of another variable that is what you actually want to test).

We do not care about checking if _custom_var is defined. We care about checking if OPENSSL_VERSION etc are defined.

Reply:
I also came across this issue when implementing the OpenSSL 3.x.x test package in #14426 . There, I changed it to
`if(DEFINED ${_custom_var} AND NOT "${${_custom_var}}" STREQUAL "" )
to also ensure that the variable is not empty

=> https://github.com/conan-io/conan-center-index/pull/14426/files#diff-cc7502af8ebc87759faacd0941ef51cbfbfc58da8a73ecf6ca9189bfc4a48e8a